charts/helix-controlplane/templates/deployment_gptscript_runner.yaml

```diff
@@ -53,3 +53,5 @@ spec:
               value: "{{ .Values.gptscript.concurrency }}"
             - name: MAX_TASKS
               value: "{{ .Values.gptscript.maxTasks }}"
+            - name: GPTSCRIPT_PROVIDER_{{ include "helix-controlplane.fullname" . | replace "." "_" | replace "-" "_" | upper }}_API_KEY
```

Comment:
This does the same thing as gptscript
* https://github.com/gptscript-ai/gptscript/blob/c5d85f1722136da53f935b4383e764975f88b436/pkg/remote/remote.go#L118
* https://github.com/gptscript-ai/gptscript/blob/c5d85f1722136da53f935b4383e764975f88b436/pkg/env/env.go#L23